pkg/kubelet/kubelet.go

```diff
@@ -2138,9 +2150,15 @@ func (kl *Kubelet) syncLoopIteration(updates <-chan kubetypes.PodUpdate, handler
 			glog.Errorf("Kubelet does not support snapshot update")
 		}
 	case <-kl.resyncTicker.C:
```

Comment:
I haven't found the need to create a goroutine in the WorkQueue yet, since we already have this busy loop here. I am not against here either, but would rather wait until we find the need to do so. What do you think?